tokio/src/runtime/scheduler/current_thread/mod.rs

```diff
@@ -459,6 +465,33 @@ impl Handle {
         handle
     }
 
+    /// Spawn a task which isn't safe to send across thread boundaries onto the runtime.
+    ///
+    /// # Safety
+    pub(crate) unsafe fn spawn_local<F>(
```

Comment:
Hi, just reading this out of interest :)

The safety section is empty here. Looks unintentional.

Reply:
Thanks for the catch!